.ai/tasks/active/ai-assist-tool-continuation/brief.md

```
@@ -0,0 +1,58 @@
+while (true) {
+  const { events, nextTurn } = AiAssist.executeClientToolTurn({
+    system, messages: baseMessages, clientTools, continuationMessages: tail
+  }).orThrow();
+  for await (const e of events) { /* observe */ }
+  const outcome = (await nextTurn).orThrow();
+  if (outcome.continuation === undefined) break;
+  tail = outcome.continuation.messages;   // REPLACE — now cumulative, no manual concat
+}
+```
+
+### Decisions to pin (with recommendations)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)